spec/defines/init_spec.rb

```diff
@@ -85,8 +85,8 @@
     it { is_expected.not_to create_group('rnelson0') }
     it { is_expected.not_to create_group('group1') }
     it { is_expected.not_to create_group('group2') }
-end
-context 'manage_groups with invalid input' do
+  end
+  context 'manage_groups with invalid input' do
```

Comment:
I dropped all of the tests that test for invalid input, since our datatypes enforce this. But that is only a good idea if the current tests are sufficient.